recipe/meta.yaml

```
@@ -18,22 +18,25 @@ build:
 
 requirements:
   build:
-    - python >=3
+    - python >=3.8, <3.10
     - pip
   run:
-    - python >=3
-    - numpy
-    - pyyaml
-    - pandas
-    - xlrd
-    - pyfai
-    - ipython
+    - python >=3.8, <3.10
     - bluesky
-    - xpdan
-    - xpdsim
+    - bluesky-darkframes
+    - frozendict
     - databroker
```

Same suggestion as pdfstream.
The previous build was failing when trying to call databroker.from_config, which no longer exists with databroker 2.x.  I think it would be a good idea to update the xpdacq and psdstream code to use databroker 2.x for a future release.  It would likely require only minimal changes.  I am happy to help out with this.

```suggestion
    - databroker <2.0.0a
```